index.html

```diff
@@ -32,49 +27,36 @@
                         <div class="search_bar">
                             <input type="text" placeholder="Search" class="typing">
                         </div>
-                        <button>
-                            <ion-icon name="search" class="search-icon"></ion-icon>
+                        <button type="button">
+                            <ion-icon name="search" class="search-icon ion-icon"></ion-icon>
                         </button>
                     </form>
                 </li>
                 <li>
-                    <buttom>
-                        <ion-icon name="mic" style="
-                        width: 24px;
-                        height: 24px;
-                        vertical-align: middle;
-                        color: inherit;
-                        outline: none;
-                        background: none;
-                        margin: 0;
-                        border: none;
-                        padding: 0;
-                        line-height: 0;
-                        cursor: pointer;
-                        cursor: pointer;
-                        color: white;"></ion-icon>
-                    </buttom>
+                    <a href="#">
+                        <ion-icon name="mic" class="ion ion-icon"></ion-icon>
+                    </a>
                 </li>
             </ul>
             <ul class="end">
                 <li class="cam">
-                    <button class="cam-button"><ion-icon name="videocam"></ion-icon></button>
+                    <button class="cam-button" type="button"><ion-icon name="videocam" class="ion-icon"></ion-icon></button>
                 </li>
                 <li>
-                    <button class="app-button"><ion-icon name="apps"></ion-icon></button>
+                    <button class="app-button" type="button"><ion-icon name="apps" class="ion-icon"></ion-icon></button>
                 </li>
                 <li>
-                    <button class="not-button"><ion-icon name="notifications"></ion-icon></button>
+                    <button class="not-button" type="button"><ion-icon name="notifications" class="ion-icon"></ion-icon></button>
                 </li>
                 <li>
-                    <button class="finger-button"><ion-icon name="finger-print"></ion-icon></button>
+                    <button class="finger-button" type="button"><ion-icon name="finger-print" class="ion-icon"></ion-icon></button>
                 </li>
             </ul>
         </header>
 
         <div id="columns">
```

Comment:
- [ ] I recommend you stick to using semantic HTML at all times. Change the div id="columns" to a section tag.